agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -24,28 +24,78 @@
 #include <gtest/gtest.h>
 #include <unistd.h>
 
-class CreateContainerTest: public ::testing::Test
+class SoftwareContainerAgentTest: public ::testing::Test
 {
 public:
-    CreateContainerTest() { }
+    SoftwareContainerAgentTest() { }
     SoftwareContainerAgent *sca;
     Glib::RefPtr<Glib::MainContext> m_context = Glib::MainContext::get_default();
     int m_preloadCount = 1;
     bool m_shutdownContainers = true;
     int m_shutdownTimeout = 2;
+    SoftwareContainerWorkspace workspace;
 
     void SetUp() override
     {
         sca = new SoftwareContainerAgent(m_context
                                          , m_preloadCount
                                          , m_shutdownContainers
                                          , m_shutdownTimeout);
```

Comment:
I know we haven't settled completely on the style, but the indentation here doesn't look like the one in the guide. Perhaps we can keep this until we've settled on a consistent guide...